### PR TITLE
Update to 1.16.5

### PR DESCRIPTION
**Pull request by official-markus-maurer**
- updated loom, yarn and fabric loader.
- fixed a one liner.

### CODE REVIEW

**Review by Sirbucket — COMMENTED** [review]
Looks good to me, game builds on 1.16.5 with phosphor.

File: gradle.properties [hunk 1]
```diff
@@ -3,9 +3,9 @@ org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
 # check these on https://modmuss50.me/fabric.html
-minecraft_version=1.16.3
-yarn_mappings=1.16.3+build.11
-loader_version=0.9.3+build.207
+minecraft_version=1.16.5
+yarn_mappings=1.16.5+build.8
```
Comment (NunzioArdi) [inline gradle.properties:7]: yarn_mappings=1.16.5+build.9 also works :)

**Comment by redbrain** [comment]
The version changes are no longer needed as 1.17 is the new latest version.
However, perhaps that one-liner should be merged in?

**Comment by amnotbananaama** [comment]
Thank you for the PR, and apologies for the very long response time. Closing this now, as the changes here have already been added to the Main branch

**Pull request closed without merging**